auth/authutils.go

```diff
@@ -29,7 +29,11 @@ func extractPayloadFromAccessToken(token string) (TokenPayload, error) {
 
 	// Decode the payload.
 	if len(tokenParts) != 3 {
-		return TokenPayload{}, errorutils.CheckErrorf("received invalid access-token")
+		return TokenPayload{}, errorutils.CheckErrorf("couldn't extract payload from Access Token.\n" +
+			"Hint: Reference and Identity Tokens are currently not supported by this functionality. " +
+			"You can use them as the password in Basic Authentication(username and password). " +
```

Comment:
@sverdlov93 -
Missing white-space before the opening parenthesis (Right after "Authentication").